patches/@types__markdown-it@14.0.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/package.json b/package.json
+index 46d7bfb79a36d8090d98d1ea1e36216137e00072..7bc1e8be5f1c5b74f52f7fa08935d2df3d19d3e2 100644
+--- a/package.json
++++ b/package.json
+@@ -27,7 +27,7 @@
+         }
+     ],
+     "main": "",
+-    "types": "index.d.ts",
++    "types": "index.d.mts",
```

Comment:
Without this patch, tsc could pass but rollup-plugin-dts would throw an error.

This patch is to make rollup-plugin-dts work. May need more investigation for the root cause.

Reply:
> Need more investigation for the root cause.

`resolvedCompilerOptions` seem to be always empty here - https://github.com/Swatinem/rollup-plugin-dts/blob/3835bfa27125db7fa5a9d071f0b93ec513d2664b/src/index.ts#L196

If we force moduleResolution bundler there then it resolves to correct files, but then the transform breaks polka by converting `export default function<T` to `declare functionexport_default<T`